html_css_dom/fetch/README.md

```diff
@@ -81,7 +81,7 @@ The arguments to `then` are option and  `catch(failureCallback)` is short for `t
 Although you will most likely already be dealing with asynchronous calls that return promises, it is possible to create your own promise with its constructor function. To do this you write new Promise and then pass in a callback function. That callback function will take in two arguments (each of which will be a function), one for resolve, and one for reject. Here's an example of what that might look like:
```

Comment:
`one for resolve, and one for reject` ... do we need to spell that out more explicitly? For example the MDN docs say:
```
We call resolve(...) when what we were doing asynchronously was successful, and reject(...) when it failed.
```
I imagine it is also very mysterious to fellows what `resolve` and `reject` actually are, like:
- what exactly passes those callbacks to the promise? where do the `resolve` and `reject` functions come from and what do they do?
- do we have to use the parameter names `resolve` and `reject`? Would `foo` and `bar` work instead?

Is it necessary or feasible to preempt some of these questions in the README?

Reply:
We definitely can talk more about those. I'm not sure if it's needed, b/c the fellows won't really be practicing building these promises.

Reply:
Fair enough, let's not go down the rabbit hole then.